### ISSUE TITLE
arcore drift when using anchors

### ISSUE THREAD

**Issue by ghost** [google-ar/arcore-android-sdk#1215]
### SPECIFIC ISSUE ENCOUNTERED
I am using the Google Earth picture as an anchor. My task is to detect the image and assign an anchor to it. Then, as the camera moves around, its pose is obtained using the getPose() function and this is sent to a server. However, I notice that as I move away from the anchor, the pose gets bad and the anchor also shifts around a bit and is no longer at its original position as was the case in the beginning. But when I come close to it again, the anchor returns to its original position on the image. Is there a solution to this problem?

### VERSIONS USED
- Android Studio: 4.1.3
- ARCore SDK for Android: 1.23.0
- Device manufacturer, model, and O/S: Oneplus 5

**Comment by devbridie** [comment]
> I am using the Google Earth picture as an anchor

Could you elaborate on what you mean by this? Are you attaching an anchor to a [Trackable](https://developers.google.com/ar/reference/java/com/google/ar/core/Trackable)?

**Comment by iAmJuan550** [comment]
Yes. I attach my anchor to this trackable object and then I grab the pose of my camera as long as I am able to track this object. However, when I move far away from this trackable object (or even if it goes out of view for a while), the AR anchor that I placed on this trackable object starts drifting and so does the pose of the camera. When I look at this trackable object again, the pose becomes normal.

I guess my question is: Is this normal behavior? I was under the impression that ARCore solved the problem of pose estimation and now we can get an accurate pose matrix for the camera. However, practically, that doesn't seem to be the case.
